public/styles/components/_tag.scss

```diff
@@ -22,6 +22,13 @@
 
 .bsds-tag {
   @include tag-default;
+  line-height: 1.5;
```

Comment:
do we need to re-define this here?

Reply:
I tried using the mixin here but the fonts weren't applying properly and I couldn't increase the specificity of tag. I could create another mixin or a variable for the line height.

Reply:
can we split line heights out of type, import them in type, and import it in tag? my biggest concern is that folks will see us setting line height in our source code without context (it's the default, aligned with wcag, wasn't applying correctly, etc), and we keep telling them not to define/override line height. if that increases scope too much, we could also split it into a separate task.